openmetadata-ui/src/main/resources/ui/src/components/EntityLineage/entityLineage.style.less

```diff
@@ -0,0 +1,74 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+@white: #fff;
+@body-color: #f8f9fa;
+@gray: #d9ceee;
+@secondary-color: #b02aac;
```

Comment:
https://github.com/open-metadata/OpenMetadata/blob/8d159851176faf96e794c393fd3260dda162c0cb/openmetadata-ui/src/main/resources/ui/src/styles/spacing.less#L14

Use this method to update variable value

Reply:
Hi, I will take care of this in the subsequent PR as I am working on UI changes for lineage